packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -62,6 +65,17 @@ export async function createConsolidatedFromConversions({
   const deduped = deduplicate({ cxId, patientId, bundle: withDups });
   log(`...done, from ${withDups.entry?.length} to ${deduped.entry?.length} resources`);
 
+  const isAiBriefFeatureFlagEnabled = await isAiBriefFeatureFlagEnabledForCx(cxId);
+  log(`isAiBriefFeatureFlagEnabled: ${isAiBriefFeatureFlagEnabled}`);
+
+  if (isAiBriefFeatureFlagEnabled) {
+    const aiBriefContent = await summarizeFilteredBundleWithAI(withDups, cxId, patientId);
+    const aiBriefFhirResource = await generateAiBriefFhirResource(aiBriefContent);
```

Comment:
at this point of the consolidated flow, each `BundleEntry<Resource>` would have already been processed to include a `request` field, like:

```
"request": {
      "method": "PUT",
      "url": "Binary/<uuid>"
  }
```

See this func for reference - [link](https://github.com/metriport/metriport/blob/eea04c8bb9645ac13cbcb62e1e27b37217c6f1b4/packages/lambdas/src/sqs-to-converter.ts#L437-L448)

The fhir converter also includes the fullUrl attribute, like:
```
"fullUrl": "urn:oid:<resource uuid>",
```
I think both of these need to be included into the `generateAiBriefFhirResource` logic. Otherwise, the consumers won't be able to insert this into their FHIR server

Reply:
We discussed request already and the fullUrl is being add by the function `buildBundleEntry` below this